src/formatter/text_formatter.ts

```diff
@@ -94,7 +94,7 @@ class TextFormatter extends Formatter {
   }
 
   chordLyricsPairLength(chordLyricsPair: ChordLyricsPair, line: Line): number {
-    const chords = renderChord(chordLyricsPair.chords, line, this.song, this.configuration.key);
+    const chords = renderChord(chordLyricsPair.chords, line, this.song, { renderKey: this.configuration.key });
```

Comment:
I might haven't noticed this earlier, but would it make sense to also as the unicode modifiers option to `TextFormatter`?